### PR TITLE
updated changes in categories service mesh/Victory Wekwa<victorywekwa…

### LINKED ISSUE #703: [Landscape] Remove SOFAMesh Duplicate from Categories
**Current State:**
The [Layer5 landscape](https://layer5.io/landscape) categories service mesh section has a duplicate of SOFAMesh.
categories

**Desired State:**
- [ ] Remove SOFAMesh duplicate
 
---
**Contributor Resources**
- [Instructions for updating the landscape](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#updating-the-service-mesh-landscape)
- Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).

### PR DESCRIPTION
**Pull request by VictoryWekwa**
…@gmail.com>

Signed-off-by: VictoryWekwa <victorywekwa@gmail.com>

**Description**

This PR fixes #703 

**Notes for Reviewers**
Remove OSFAMesh Duplicate from Categories

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by PluckyPrecious** [comment]
Oh my! 😮 @VictoryWekwa made a PR already.

File: _data/landscape/meshes.yml [hunk 1]
```diff
@@ -287,8 +287,8 @@
   tracing: "<i><a class='blue-grey-text' href='https://blog.turbinelabs.io/turbine-labs-is-shutting-down-and-our-team-is-joining-slack-2ad41554920c'>Project shutdown</a></i>"
   encryption: "<i><a class='blue-grey-text' href='https://blog.turbinelabs.io/turbine-labs-is-shutting-down-and-our-team-is-joining-slack-2ad41554920c'>Project shutdown</a></i>"
 
-- name: SOFAMesh
-  desc: A solution for large-scale Service Mesh based on Istio.
+- name: Open Service Mesh
+  desc: Open Service Mesh (OSM) is a lightweight and extensible cloud native service mesh.
```
Comment (leecalcote) [inline _data/landscape/meshes.yml:291]: @VictoryWekwa Thanks! You can go ahead and delete lines to 92 through 301 as well. https://github.com/layer5io/layer5/pull/704/files#diff-c960221cf975c9a39633140c3e0f3d72R292-R301

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _data/landscape/meshes.yml (hunk 1 above)
Comment (PluckyPrecious) [reply _data/landscape/meshes.yml:291]: @leecalcote do you mean 92 or 192? 😮

Comment (VictoryWekwa) [reply _data/landscape/meshes.yml:291]: Okay. Will make the necessary corrections and create a pull request.
Should be 292 through 301.

**Review by Nikhil-Ladha — APPROVED** [review]

**Review by Nikhil-Ladha — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you for this review, @PluckyPrecious 

And successful deduplication @VictoryWekwa

**Pull request merged**